src/main/java/io/coala/jetbrains/utils/Notifier.java

```diff
@@ -0,0 +1,45 @@
+package io.coala.jetbrains.utils;
+
+import com.intellij.notification.*;
+import com.intellij.openapi.actionSystem.AnActionEvent;
+import com.intellij.openapi.application.ApplicationManager;
+import com.intellij.openapi.ui.MessageType;
+import com.intellij.openapi.wm.ToolWindowManager;
+import org.apache.commons.lang.exception.ExceptionUtils;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public final class Notifier {
+    private static final String title = "coala";
+    private static final NotificationGroup NOTIFICATION = new NotificationGroup("coala", NotificationDisplayType.BALLOON, true);
```

Comment:
`129` Characters long, max is `120`.

Reply:
Lines flowing over 120, to about 125-130 have been kept as it is to improve code readability.